modules/ROOT/pages/uninstall.adoc

```diff
@@ -0,0 +1,161 @@
+
+[source,shell]
+----
+astra nuke
+----
+
+When prompted, type kbd:[y] and press kbd:[Enter] to proceed with the uninstallation.
+
+The `astra nuke` command deletes the {product} home directory and optionally the configuration files (`xref:ROOT:manage-cli.adoc#file-locations[~/.astra]` and `xref:ROOT:manage-cli.adoc#file-locations[~/.astrarc]`).
+It also lists any shell profile containing references to the `astra` command.
+Delete these references manually, if desired.
+--
+
+Manual removal::
++
+--
+. Optional: Delete the `xref:ROOT:manage-cli.adoc#file-locations[.astrarc]` file:
++
+[source,shell]
+----
+rm $(astra config path -p)
+----
++
+Only do this if you don't want to preserve your xref:ROOT:manage-cli.adoc#manage-configuration-profiles[configuration profiles].
+
+. Delete the xref:ROOT:manage-cli.adoc#file-locations[{product} home directory]:
++
+[source,shell]
+----
+rm -rf $(astra config home path -p)
+----
+
+. Delete the `astra` binary if it still exists:
++
+[source,shell]
+----
+rm -f $(which astra)
+----
+
+. Remove any references to the `astra` command from your shell profile.
+For example, from `~/.zprofile` or `~/.bash_profile`.
+--
+=====
+
+Windows::
++
+Select one of the following methods to uninstall the {product} on Windows:
++
+[tabs]
+=====
+Uninstaller (recommended)::
++
+--
+Only use this method if you installed the {product} using the xref:ROOT:install.adoc[scripted installer].
+
+. Optional: Delete the `xref:ROOT:manage-cli.adoc#file-locations[.astrarc]` file:
++
+[source,powershell]
+----
+rm -f $(astra config path -p)
+----
++
+Only do this if you don't want to preserve your xref:ROOT:manage-cli.adoc#manage-configuration-profiles[configuration profiles].
+
+. Go to *Start > Settings  > Apps > Installed apps*.
+In the list of apps, find the {product}, select icon:ellipsis-vertical[name="More"], and then select *Uninstall*.
++
+[NOTE]
+====
+The steps to locate and uninstall applications may vary depending on your version of Windows.
+For more information, see the https://support.microsoft.com/en-us/windows/uninstall-or-remove-apps-and-programs-in-windows-4b55f974-2cc6-2d2b-d092-5905080eaf98[Microsoft Windows documentation].
+====
+--
+
+`astra nuke` command::
++
+--
+Use the `xref:commands:astra-nuke.adoc[]` command to uninstall the {product}:
+
+[source,powershell]
+----
+astra nuke
+----
+
+When prompted, type kbd:[y] and press kbd:[Enter] to proceed with the uninstallation.
+
+The `astra nuke` command deletes the {product} home directory and optionally the configuration files (`xref:ROOT:manage-cli.adoc#file-locations[~/.astra]` and `xref:ROOT:manage-cli.adoc#file-locations[~/.astrarc]`).
+
+You must remove `astra` from your system `PATH` manually, if desired.
+For more information, see the https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables[Microsoft PowerShell documentation].
```

Comment:
Rearranged to align with how it's presented for Homebrew,  and I have the same question here about the word "optionally".

```suggestion
. Use the `xref:commands:astra-nuke.adoc[]` command to uninstall the {product} and delete the {product} home directory and configuration files (`xref:ROOT:manage-cli.adoc#file-locations[~/.astra]` and `xref:ROOT:manage-cli.adoc#file-locations[~/.astrarc]`):
+
[source,powershell]
----
astra nuke
----
+
When prompted, type kbd:[y] and press kbd:[Enter] to proceed with the uninstallation.

. Optional: Manually remove `astra` from your system `PATH`.
For more information, see the https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables[Microsoft PowerShell documentation].
```